src/js/import.js

```diff
@@ -195,8 +220,17 @@
         $(".import-extraHeaders").each(function() {
           selected.push($(this).val());
         });
-        $(".import-leave").each(function() {
-          selected.push($(this).val());
+        $(".import-intervention").each(function() {
+         selected.push($(this).val());
+        });
+        $(".import-effect").each(function() {
+         selected.push($(this).val());
+        });
```

Comment:
You could also skip jquery and use standard:

`Array.from(document.querySelectorAll('.import-serp-select-wrapper > select')).map(...)`

But I don't know if `Array.from` is safe to use...

`querySelectorAll` returns a `NodeList` or something, which isn't treated as an array by javascript. It is possible to hack around using `Array.prototype.map` but it's up to you to experiment!